### PR TITLE
Rolling Number Apply

### PR DESCRIPTION
**Pull request by revonateB0T**

### CODE REVIEW

File: app/src/main/kotlin/com/hippo/ehviewer/client/parser/ArchiveParser.kt [hunk 1]
```diff
@@ -62,7 +62,7 @@ object ArchiveParser {
         val funds = maybeFunds ?: PATTERN_CURRENT_FUNDS.find(body)!!.groupValues.run {
             val fundsGP = ParserUtils.parseInt(get(1), 0)
             val fundsC = ParserUtils.parseInt(get(2), 0)
-            Funds("%,d".format(fundsGP), "%,d".format(fundsC))
+            Funds(fundsGP, fundsC)
```
Comment (FooIbar) [inline app/src/main/kotlin/com/hippo/ehviewer/client/parser/ArchiveParser.kt:65]: It's less readable without separators.

Comment (revonateB0T) [reply app/src/main/kotlin/com/hippo/ehviewer/client/parser/ArchiveParser.kt:65]: Well, how can I add separators to RollingNumber?

Comment (revonateB0T) [reply app/src/main/kotlin/com/hippo/ehviewer/client/parser/ArchiveParser.kt:65]: Ok, I will add it

Comment (revonateB0T) [reply app/src/main/kotlin/com/hippo/ehviewer/client/parser/ArchiveParser.kt:65]: Done

**Review by FooIbar — DISMISSED** [review]

File: app/src/main/kotlin/com/hippo/ehviewer/client/parser/HomeParser.kt [hunk 2]
```diff
@@ -30,7 +30,7 @@ object HomeParser {
         FundsRegex.find(body)?.groupValues?.run {
             val fundsC = ParserUtils.parseInt(get(1), 0)
             val fundsGP = ParserUtils.parseInt(get(2), 0) * 1000
-            return Funds("%,d+".format(fundsGP), "%,d".format(fundsC))
+            return Funds(fundsGP, fundsC)
```
[the remark this answers is not in the document]
Comment (revonateB0T) [reply app/src/main/kotlin/com/hippo/ehviewer/client/parser/HomeParser.kt:33]: I think we could divide by 1000 and save some space

[the remark this answers is not in the document]
Comment (revonateB0T) [reply app/src/main/kotlin/com/hippo/ehviewer/client/parser/HomeParser.kt:33]: Done

**Review by FooIbar — APPROVED** [review]

**Pull request merged**